include/Application.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+** Project FileShare, 2022
+**
+** Author Léo Lhuile
+**
+** Started on  Sat Jan 15 21:08:44 2022 Léo Lhuile
+** Last update Wed Sep 14 22:26:27 2022 Léo Lhuile
+**
+** Application.hpp : Class to manage the application.
+*/
+
+#pragma once
+
+#include <TGUI/TGUI.hpp>
+#include <TGUI/Backend/SFML-Graphics.hpp>
+
+namespace FileShare::GUI {
+    #define ANIMATION_DURATION 250
```

Comment:
let's use constants (const or constexpr static) instead of macros. Macros leak outisde of namespaces.